CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Change Log
 
+## [1.2.0rc1] - 2022-08-16
+
+### Added
+- Virtual environments can now be created even with empty project name ([#5856](https://github.com/python-poetry/poetry/pull/5856))
```

Comment:
I think this change belongs to `Fixed` (or `Changed`) rather than `Added`?

Reply:
I wasn't sure about it. IMO an empty project name isn't useful and therefor it was never considered as supported. Now it is, that's why I put it into this section. But if there are strong feelings against this, I can change it.

Reply:
No strong feelings, I'm fine with keeping it here.